src/main/scala/nl/knaw/dans/easy/fedoratobag/versions/Resolver.scala

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) 2020 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.fedoratobag.versions
+
+import scalaj.http.Http
+
+import scala.util.{ Success, Try }
+
+case class Resolver() {
+
+  def getDatasetId(id: String): Try[String] = {
+    id.slice(0, 3) match {
+      case "eas" => Success(id)
+      case "urn" => resolve(s"http://www.persistent-identifier.nl/?identifier=$id")
```

Comment:
this is a shortcut, there are other urn: links. but how often they appear in the easy-datasets i don't know

Reply:
This is called with stripped IDs https://github.com/DANS-KNAW/easy-fedora-to-bag/blob/82a77d28aebb7214cbf1c79d44297d00477af813/src/main/scala/nl/knaw/dans/easy/fedoratobag/versions/VersionInfo.scala#L75-L81